superset/db_engine_specs/base.py

```diff
@@ -538,7 +538,9 @@ def get_schema_names(cls, inspector: Inspector) -> List[str]:
         return sorted(inspector.get_schema_names())
 
     @classmethod
-    def get_table_names(cls, inspector: Inspector, schema: Optional[str]) -> List[str]:
+    def get_table_names(
+        cls, database, inspector: Inspector, schema: Optional[str]
```

Comment:
nit: can we add a type for database? comment applies elsewhere

Reply:
Will do!